replay/backend/models/TabBackend.ts

```diff
@@ -1,16 +1,20 @@
-import { BrowserView, app } from 'electron';
+import { BrowserView } from 'electron';
 import { URL } from 'url';
-import * as Path from 'path';
 import { v1 as uuidv1 } from 'uuid';
 import * as http from 'http';
 import Window from './Window';
 import generateContextMenu from '../menus/generateContextMenu';
 import ITabLocation, { InternalLocations } from '~shared/interfaces/ITabLocation';
 import { INTERNAL_BASE_URL } from '~shared/constants/files';
-import ITabMeta from '~shared/interfaces/ITabMeta';
 import ReplayApi from '~backend/ReplayApi';
+import IRectangle from '~shared/interfaces/IRectangle';
+import Rectangle = Electron.Rectangle;
+import Application from '~backend/Application';
+import ipcMain = Electron.ipcMain;
+import ICommandResult from '~shared/interfaces/ICommandResult';
+import { ITick } from '~shared/interfaces/ISaSession';
 
-const domReplayerScript = Path.join(app.getAppPath(), '..', 'injected-scripts/domReplayer.js');
+const domReplayerScript = require.resolve('../../injected-scripts/domReplayer');
```

Comment:
Why the change? Just curious.

Reply:
It didn't work with the new build directories when using dev mode